.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+
+node_js:
+  - '14'
+
+script:
+  - npm run prettier:check
+  - npm run lint:check
+  - npm run test:coveralls
+
+notifications:
```

Comment:
How do we setup notification. Need suggestion here.

Reply:
For the addons-linter the travis notifications are being sent to slack, on web-ext the travis notification are still configured to go to irc (which we don't use anymore, and so it is kind of /dev/null :-P).

Personally I think that we can leave the travis notification disabled (at least for now).